carbon-design-system/carbon
# refactor(project): move bundler to cli

This PR introduces an internal refactor to merge `@carbon/bundler` and `@carbon/cli`. As a result, all commands in bundler are now in carbon-cli and `@carbon/bundler` has been removed from the project. This PR also removes commands from bundler that were no longer used in the project.

#### Changelog

**New**

**Changed**

- Update all instances of `bundler` to be `carbon-cli`
- Move bundler commands to `packages/cli`

**Removed**

- `packages/bundler` has been removed

#### Testing / Reviewing

- Verify the build passes as expected
- Verify packages using `carbon-cli bundle` compile as expected
